_data/gremien.yml

```diff
@@ -2,38 +2,40 @@ vorstand:
   - name: "Geschäftsführender Vorstand"
     email: "vorstand@pep-dortmund.org"
     people:
-      - name: "Kevin Schmitz"
+      - name: "Christian Arauner"
         role: "1. Vorsitzender"
-        image: "/images/vorstand/kevinS.jpg"
-      - name: "Karl Schiller"
+        image: "/images/vorstand/christian_Arauner.jpg"
+      - name: "Rene-Marcel Lehner"
         role: "2. Vorsitzender"
-        image: "/images/vorstand/karl.jpg"
-      - name: "Lars Bollmann"
+        image: "/images/vorstand/rene.jpg"
+      - name: "Anno Knierim"
         role: "Finanzreferent"
-        image: "/images/vorstand/lars.jpg"
+        image: "/images/vorstand/anno.jpg"
 
   - name: "Absolventenfeier"
     email: "koordination@pep-dortmund.org"
```

Comment:
we should change mail adresses according to the actual name of the job

Reply:
done